services/api/service.go

```diff
@@ -1398,7 +1403,7 @@ func (api *RelayAPI) handleSubmitNewBlock(w http.ResponseWriter, req *http.Reque
 	}
 
 	// Prepare the response data
-	getHeaderResponse, err := BuildGetHeaderResponse(payload, api.blsSk, api.publicKey, api.opts.EthNetDetails.DomainBuilder)
+	getHeaderResponse, err := BuildDummyGetHeaderResponse(payload, api.blsSk, api.publicKey, api.opts.EthNetDetails.DomainBuilder)
```

Comment:
what is the value of building a dummy here ?

Reply:
maybe just the name Dummy makes it sound like a test. maybe Empty is better ?

Reply:
Just wanted to make it abundantly clear this is not producing a valid payload, just the necessary bits that the Redis stuff needs. Empty works too, makes sense

Reply:
done